flake.nix

```diff
@@ -175,6 +184,14 @@
               "/tests/(.+)\\.ncl$"
             ];
           };
+
+          fresh-grammar = {
+            name = "fresh-grammar";
+            enable = true;
+            description = "Check that the lalrpop output is up-to-date.";
+            files = "\\.lalrpop";
+            entry = "${check-fresh-grammar}/bin/check-fresh-grammar";
```

Comment:
It's a detail, but would a `lib.getExe` or the like work here, so that we don't even have to rely on the actual file Nix generates?